### ISSUE TITLE
`Lint/EmptyBlock` should not apply to empty lamdas

### ISSUE THREAD

**Issue by dvandersluis** [rubocop-hq/rubocop#9015]
It is often useful to have an empty lambda as a callable placeholder (for instance to stub out a callable or as an initial value in dynamic function composition. `Lint/EmptyBlock` flags these, but there isn't really a better way to express these (`-> { nil }` is not better here IMO). This should apply to both `lambda` and `->`.

--------

## Expected behavior

Rubycop should not register any offenses about this code:

```ruby
expect(Foo).to receive(:callable).and_return(-> {})
```

## Actual behavior

Rubocop registers an offense.

```
Offenses:

spec/test_spec.rb:1:46: W: Lint/EmptyBlock: Empty block detected.
expect(Foo).to receive(:callable).and_return(-> {})
                                             ^^^^^
```

## RuboCop version

```
1.2.0 (using Parser 2.7.2.0, rubocop-ast 1.1.1, running on ruby 2.6.2 x86_64-darwin18)
  - rubocop-performance 1.8.1
  - rubocop-rails 2.8.1
  - rubocop-rspec 2.0.0
```

**Comment by gPrado** [comment]
Is there any reason to ignore empty lambdas but register offenses for empty procs (i.e. `Proc.new {}` or `proc {}`)?
The usage as a placeholder is very similar to the mentioned for lambdas, but when arguments must be ignored.

**Comment by dvandersluis** [comment]
@gPrado makes sense, can you make a new issue?
